Gems/Multiplayer/Code/Source/NetworkEntity/EntityReplication/PropertyPublisher.h

```diff
@@ -66,35 +100,45 @@ namespace Multiplayer
         EntityReplicatorState GetReplicatorState() const;
 
         //! Check if we have data to send
-        bool HasUpdateEntityRecord();
+        bool HasEntityChangesToSend();
 
         //! Phase 1, setup of the record
-        bool PrepareAddEntityRecord();
-        bool PrepareRebaseEntityRecord();
-        bool PrepareUpdateEntityRecord();
-        bool PrepareDeleteEntityRecord();
+        void PrepareAddEntityRecord(NetBindComponent* netBindComponent);
```

Comment:
Oh.. perhaps I got this wrong and an add record is actually a create record, which includes the prefabentityId?

Reply:
Hmmm. Let me poke at this a bit and make sure I get it right before I comment it. I *think* it's marking _every_ networked property as changed indiscriminately, regardless of whether or not it differs from the prefab. I assume it's called an "add" record because you'd typically use that at replicated creation time, though as you pointed out, it also gets used when too many updates get lost, so it's not _literally_ an add or create.

Reply:
It might just be everything, since that's easier than diffing against the prefab version and probably safer too (in the case where something changed away from default and then got restored back to a default value later).